docs/en/week04/practicum04.sbv

```diff
@@ -1,379 +1,379 @@
-wait for a sec ready yes no shake your
+Okay. Am I waiting a little bit? Let me
+wait for a sec. Ready?? Yes? No? shake your
 
 0:02:06.450,0:02:12.150
-head okay fantastic alright so we were
+head. Okay fantastic, alright so were
 talking last time we had a neural
 
 0:02:12.150,0:02:15.270
 network with the input on the bottom
-then we had like an affine
+then we had an affine
 
 0:02:15.270,0:02:18.510
-transformation then we have a hidden
-layer right so I'm gonna just write the
+transformation, then we have a hidden
+layer right. So I'm gonna just write the
 
 0:02:18.510,0:02:23.280
-first equation we're gonna have that my
-hidden layer and since I'm writing on
+first equation. We're gonna have that my
+hidden layer, and since I'm writing
 
 0:02:23.280,0:02:26.970
-with a pen can you see anything yeah
+with a pen, can you see anything? yeah?
 so since I'm writing with a pen I'm
 
 0:02:26.970,0:02:30.360
 going to be putting a underscore
 underneath the variable in order to
 
 0:02:30.360,0:02:35.120
-indicate it's a vector okay that's how I
-write vectors so my H is gonna be a
+indicate it's a vector. Okay? that's how I
+write vectors. So my H is gonna be a
 
 0:02:35.120,0:02:42.660
-nonlinear function f apply to my Z and
-Zed is gonna be my linear input
+nonlinear function f applied to my z and
+z is going to be my linear input,
 
 0:02:42.660,0:02:46.230
-okay the linear when the output of the
-affine transformation so in this case
+the output of the
+affine transformation, so in this case
 
 0:02:46.230,0:02:54.989
-I'm gonna be writing here Z is going to
-be equal to my matrix a times X okay we
+I'm gonna be writing here z is going to
+be equal to my matrix A times x. We
 
 0:02:54.989,0:03:00.630
 can imagine there is no bias in this
-case it's generic enough because we can
+case, it's generic enough because we can
 
 0:03:00.630,0:03:06.030
 include it bias inside the matrix and
-have the first item of the X equal to B
+have the first item of the x equal to 
 
 0:03:06.030,0:03:18.360
-1 so if this X here belongs to RN and is
-dead here belongs to RM first question
+1. So if this x here belongs to R n and this
+z here belongs to R m, first question:
 
 0:03:18.360,0:03:24.989
-what is the size of this matrix okay
-fantastic right so this matrix here is
+what is the size of this matrix? Okay,
+fantastic, so this matrix here is
 
 0:03:24.989,0:03:30.360
-gonna be our M times n you have as many
-rows as the dimension where you should -
+gonna be our m times n, you have as many
+rows as the dimension where you shoot to
 
 0:03:30.360,0:03:34.200
-and you have as many columns the
-dimension where you're shooting from ok
+and you have as many columns as the
+dimension where you're shooting from, ok?
 
 0:03:34.200,0:03:39.930
-all right so let's expand this one right
+All right so let's expand this one, 
 so this matrix here is gonna be equal to
 
 0:03:39.930,0:03:49.549
 what do you have a 1 1 a 1 2 so on until
```

Comment:
@Atcold What do you think about  the caption for subscripts? Should we leave it like this or perhaps change it to something more symbolic like a_(1,1)   a_(1,2) etc.

Reply:
I was thinking about it as the video progressed and wondering how it was being captioned in other videos. I think a_(1,1) would be more intuitive than a 1 1.

Reply:
I agree with a_{1, 1}! Let's update it!